<?xml version="1.0" encoding="UTF-8"?>
<?bpmo version="12.1.3.0.0.00.00" build="02.10" fullName="Exception.JInternal_BizException" modifiers="[exception]"?>
<!--########################################################################################-->
<!--####                                                                                ####-->
<!--####          THIS IS AN AUTO GENERATED FILE. PLEASE DO NOT MANUALLY MODIFY         ####-->
<!--#### CHANGES TO THIS FILE MAY PRODUCE UNEXPECTED BEHAVIOR AND WOULD BE OVERWRITTEN  ####-->
<!--####                                                                                ####-->
<!--########################################################################################-->
<xs:schema targetNamespace="http://xmlns.oracle.com/bpm/bpmobject/Exception/JInternal_BizException" elementFormDefault="qualified" xmlns="http://xmlns.oracle.com/bpm/bpmobject/Exception/JInternal_BizException"  xmlns:xs="http://www.w3.org/2001/XMLSchema"  xmlns:bpmo="http://xmlns.oracle.com/bpm/bpmobject/" >
    <xs:complexType name="JInternal_BizExceptionType">
        <xs:sequence>
            <xs:element name="errorInfo" nillable="true" type="xs:string"/>
        </xs:sequence>
    </xs:complexType>
    <xs:element name="JInternal_BizException" type="JInternal_BizExceptionType"/>
</xs:schema>
